### ISSUE TITLE
Undefined variable: etagMatches and timeMatches

### ISSUE THREAD

**Issue by bichomen** [cakephp/cakephp#12672]
Hello, I recently updated PHP to the PHP 7.3 version and these two advents have appeared. How do I solve it?

Thank you

CakePHP version: Is the 3.6

////////////////////////////////////////////////////////////////////////////////////////////////////
// +--------------------------------------------------------------------------------------------+ //
// CakePHP Version
//
// Holds a static string representing the current version of CakePHP
//
// CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
// Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
//
// Licensed under The MIT License
// Redistributions of files must retain the above copyright notice.
//
// @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
// @link          https://cakephp.org
// @since         CakePHP(tm) v 0.2.9
// @license       https://opensource.org/licenses/mit-license.php MIT License
// +--------------------------------------------------------------------------------------------+ //
////////////////////////////////////////////////////////////////////////////////////////////////////

Thank you

```
Notice (8): compact(): Undefined variable: etagMatches [CORE/src/Http/Response.php, line 2051]

Notice (8): compact() [<a href='https://secure.php.net/function.compact'>function.compact</a>]: Undefined variable: timeMatches [CORE/src/Http/Response.php, line 2051]
```

**Comment by markstory** [comment]
What version of CakePHP are you using?

**Comment by bichomen** [comment]
update info

**Comment by ADmad** [comment]
Already fixed in https://github.com/cakephp/cakephp/commit/249cbc3d3a00e9eca931f44ee6990312f156e6cc on `master`. It will be available in next 3.6 bugfix release.

**Comment by infinitbility** [comment]
Explain Here Step by Step to solve the issue

https://infinitbility.com/undefined-variable-timematches-and-etagmatches-cakephp
